service/src/main/java/uk/gov/hmcts/reform/fpl/model/HearingBooking.java

```diff
@@ -43,4 +50,13 @@ public boolean hasDatesOnSameDay() {
     public boolean startsAfterToday() {
         return startDate.isAfter(ZonedDateTime.now(ZoneId.of("Europe/London")).toLocalDateTime());
     }
+
+    public boolean hasCMOAssociation() {
+        return caseManagementOrderId != null;
+    }
+
+    public String toLabel(String dateFormat) {
```

Comment:
I don't massively mind it here, but I don't think it's an ideal place - the reason is different "pages" might expect different representation of hearingBooking label. But it's fine for the starters.

We have got Welsh language looming as well, having English text scattered through the codebase is far from ideal.

Reply:
Fair enough, was wondering how generic to make it. Are you happy for me to leave it here? I think it is better than the alternative that was just merged in with notice of proceedings which has something very similar in the hearing booking service but happy to change if you think there is a better place.

Reply:
Happy to leave it here for now.